docs/contributing/index.md

```diff
@@ -116,18 +125,60 @@ The benefits of this approach are that the link won't break if
 
 ### Images
 
+> A picture is worth a thousand words
+
+We encourage the usage of images to improve clarity and understanding. You can use **screenshots** or **diagrams**.
+
 Images are stored in the `docs/images` directory.
 
-* there are sub-directories in the `docs/images` path - create a new sub-directory for your images if appropriate
-* choose image and path names that make sense - `screenshot.png` is not a great file name. Neither is `PX-202502025-imgx.png`.
+* create a new sub-directory for your images if appropriate
+* choose a path and file name that hint what the image is about - neither `screenshot.png` nor `PX-202502025-imgx.png` are great names.
 
-!!! tip
-    When providing a screenshot, do you need to show the whole screen, or just part of one window?
+!!! warning
+    Keep the size of your images to a minimum because we want to keep an overall lightweight repository.
 
-    Cropping the image will decrease file size, and might also draw the readers attention to the most relevant information.
 
-!!! tip
-    Do you need a screenshot, or can a text description also work?
+#### Screenshots
```

Comment:
I think `screenshot` could be added to `.github/actions/spelling/allow.txt`. Wikipedia seems to use "screenshot" as well (not "screen shot"), so the spell checker is wrong here.

Reply:
adding `codeowners` and `screenshots`. I wonder which tool/config are you using to keep the alphabetical order? my sort yields something different so I just placed them manually

Reply:
Ah, fair question. Emacs' `sort-lines` :upside_down_face: I suppose mine sorts caps before lower case, whereas yours maybe mixes them?